backend/src/main/java/com/htecgroup/skynest/filter/CustomAuthenticationFilter.java

```diff
@@ -80,15 +79,13 @@ protected void successfulAuthentication(
       Authentication authentication)
       throws IOException, ServletException {
 
-    User user = (User) authentication.getPrincipal();
+    LoggedUserDto user = (LoggedUserDto) authentication.getPrincipal();
+    List<String> authorities = user.getRoleNames();
 
-    List<String> authorities =
-        user.getAuthorities().stream()
-            .map(GrantedAuthority::getAuthority)
-            .collect(Collectors.toList());
+    Map<String, String> payload = Map.of("uuid", user.getUuid().toString());
     String token =
         JwtUtils.generate(
-            user.getUsername(), JwtUtils.ACCESS_TOKEN_EXPIRATION_MS, "roles", authorities);
+            user.getUsername(), payload, JwtUtils.ACCESS_TOKEN_EXPIRATION_MS, "roles", authorities);
```

Comment:
I am just thinking at loud :) 

Is it possible to combine, username, and this uuid that is set in the map, into a custom object and send that object in the JwtUtils.generate method? Does that makes sense ?

Reply:
I'll take a look